models/HrSchema.ts

```diff
@@ -0,0 +1,45 @@
+import { Schema, model } from 'mongoose';
+
+const HrSchema = new Schema(
+  {
+    email: {
+      type: String,
+      required: true,
+      max: 50,
+      unique: true,
+      match: [/^(.+)@(.+)$/, 'Invalid email'],
+    },
+    fullName: {
+      type: String,
+      required: true,
+    },
+    company: {
+      type: String,
+      required: true,
+    },
+    maxReservedStudents: {
+      type: Number,
+      min: 0,
+      max: 999,
+      default: 0,
+    },
+    password: {
+      type: String,
+      required: true,
+      bcrypt: true,
+    },
+    token: {
+      type: String,
+      default: '',
+    },
+    role: {
+      type: String,
+      enum: ['Admin', 'Kursant', 'HR'],
+      default: '',
+    },
+    users: [String],
+  },
+  { timestamps: true, versionKey: false }
+);
+
```

Comment:
pytanie jeszcze czy nie powinnismy dodac pola związanego z limitami dla hra, by mogł wybrac tylko okreslona liczne kursantów?

Reply:
a to nie będzię to samo co maxReservedStudents? Czyli podczas tworzenia HR przez formularz już sztywno określamy ile dany HR może wybrać kursantów?

Reply:
maxReservedStudents to jest właśnie to pole i jest wg dokumentacji od 0 do 999